api/metadata/permissions.py

```diff
@@ -0,0 +1,77 @@
+from contextlib import suppress
+
+from rest_framework.permissions import IsAuthenticated
+
+from metadata.models import MetadataField
+from organisations.models import Organisation
+
+
+class MetadataFieldPermissions(IsAuthenticated):
+    def has_permission(self, request, view):
+        if not super().has_permission(request, view):
+            return False
+
+        # list is handled by the view
+        if view.action == "list" or view.detail:
+            return True
+
+        if view.action == "create":
+            with suppress(Organisation.DoesNotExist):
+                organisation_id = request.data.get("organisation")
```

Comment:
Shouldn't we get this from `self.kwargs` rather than the request data? Can we remove `organisation` from the request data?